.github/workflows/linters.yml

```diff
@@ -10,15 +10,15 @@ concurrency:
 
 jobs:
   nph:
-    name: NPH
+    name: Linters
```

Comment:
or any other suggestion?
but NPH doesn't make sense now that we are also running other checks here